lib/ui/widgets/message/group/group_message_tile.dart

```diff
@@ -1,7 +1,30 @@
+import 'package:auto_size_text/auto_size_text.dart';
 import 'package:flutter/material.dart';
+import 'package:virt_mnc/app/sender_entity.dart';
+import 'package:virt_mnc/app/user_data.dart';
 
 class GroupMessageTile extends StatefulWidget {
-  const GroupMessageTile({super.key});
+  GroupMessageTile({
+    super.key,
+    this.id,
+    this.messageStr = "Uninitialised messageStr",
+    required this.sender,
+    required this.isMe,
+  }) {
+    if (isMe == true) {
```

Comment:
Simplify if statement to 
```suggestion
    if (isMe) {
```

Reply:
Ok